src/module/Ui/templates/entity/repository/compare-revision.twig

```diff
@@ -50,26 +49,14 @@
                             {% if oldField %}
                                 {{ diff().markdown(oldField, field.getValue()) }}
                             {% else %}
-                                {{ (revisionIsOry ? oryRenderer() : markdown()).toHtml(field.getValue()) }}
+                                {{ (fieldIsOry ? oryRenderer() : markdown()).toHtml(field.getValue()) }}
                             {% endif %}
                         </div>
                         <div class="tab-pane fade" id="old{{ field.getName() }}">
-                            {% if compareRevisionIsOry %}
-                            <div class="editable" data-edit-type="ory">
-                                {{ oryRenderer().toHtml(compareRevision.get(field.getName())) }}
-                            </div>
-                            {% else %}
-                                {{ markdown().toHtml(compareRevision.get(field.getName())) }}
-                            {% endif %}
+                            {{ (fieldIsOry ? oryRenderer() : markdown()).toHtml(compareRevision.get(field.getName())) }}
```

Comment:
You need a second boolean for this. The problem here is, that we're in the process of switching from one editor to the next, so one revision might already be created with the (new) ory editor and the other still be with the markdown editor. So I think something like the folowing would be needed:
```
{% set compareFieldIsOry = isOryEditorFormat(compareRevision.get(field.getName())) %}
{{ (compareFieldIsOry ? oryRenderer() : markdown()).toHtml(compareRevision.get(field.getName())) }}
```
Or something like this.